templates/statisticItem.hbs

```diff
@@ -0,0 +1,4 @@
+<div class="statistics__item">
```

Comment:
Can you change this `div` for a `p`? It will improve accessibility.

Reply:
done! Thank you :)